recipes/pyomo/meta.yaml

```diff
@@ -0,0 +1,166 @@
+{% set version = "4.3.11377" %}
+
+package:
+  name: pyomo
+  version: {{ version }}
+
+source:
+  fn: Pyomo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
```

Comment:
So, they really messed up PyPI. It took a bit of digging and help from other people to figure out the cause. Basically because the old PyPI will be replaced with Warehouse, we have to deal and change our URLs.

Basically, change the URL here to this `https://pypi.io/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz`.